mlrun/db/httpdb.py

```diff
@@ -2770,6 +2770,74 @@ def trigger_migrations(self) -> Optional[schemas.BackgroundTask]:
             )
         return None
 
+    def dataschema_to_list(self):
+        """Provide data schema (feature store schemas) of MLRun projects as collection.
+
+        Examples::
+
+            import mlrun.db.httpdb as mldb
+            import json
+
+            # connection to solution
+            db = mldb.HTTPRunDB('http://localhost:8080')
+            # or db = mldb.HTTPRunDB()
```

Comment:
```suggestion
            db = mlrun.get_run_db()
```

This utility function is better to use than the specific calls used - it already identifies the URL of the target and returns the `HTTPDB` instance needed.